src/models/monitoringReviewGrantee.js

```diff
@@ -0,0 +1,93 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class MonitoringReviewGrantee extends Model {
+    static associate(models) {
+      models.MonitoringReview.hasMany(
+        models.MonitoringReviewGrantee,
+        {
+          foreignKey: 'reviewId',
+          targetKey: 'reviewId',
+          as: 'monitoringReviewGrantees',
+        },
+      );
+
+      models.MonitoringReviewGrantee.belongsTo(
+        models.MonitoringReview,
+        {
+          foreignKey: 'reviewId',
+          targetKey: 'reviewId',
+          as: 'monitoringReview',
+        },
+      );
+
+      models.Grant.hasMany(
+        models.MonitoringReviewGrantee,
+        {
+          foreignKey: 'number',
+          targetKey: 'grantNumber',
+          as: 'monitoringReviewGrantees',
```

Comment:
I'm not sure these are configured correctly. When I do an include like so:

```js
include: [{
      model: Grant,
      attributes: ['id', 'recipientId', 'regionId', 'number'],
      where: { regionId },
      required: true,
      as: 'grants',
      include: [
        {
          model: MonitoringReviewGrantee,
          attributes: ['id', 'grantNumber', 'reviewId'],
          required: true,
          as: 'monitoringReviewGrantees',
        }
    }]
```

I get the following error:
```
column grants->monitoringReviewGrantees.number does not exist

Reply:
fixing now